flixel/util/FlxTimer.hx

```diff
@@ -273,4 +273,14 @@ class FlxTimerManager extends FlxBasic
 	{
 		FlxArrayUtil.clearArray(_timers);
 	}
-}
\ No newline at end of file
+	
+	/**
+	 * Calls the onComplete function of all timers, then removes them from the timer manager.
+	 */
+	public function completeAll():Void
+	{
+		for (timer in _timers)
+			timer.onComplete(timer);
```

Comment:
This completely bypasses all the timer logic. The user might check properties of the timer on the callback (one of the reasons why it's passed as an argument), and they would be inconsistent with how they would look like on normal completion.